### PR TITLE
Add log_overhead_s to all structured log messages

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #131492

This is to measure the overhead of collecting data that will be run
in structured logs.

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

### CODE REVIEW

File: torch/_logging/_internal.py [hunk 1]
```diff
@@ -1084,6 +1085,7 @@ def trace_structured(
     # trace_log never propagates and is ALWAYS DEBUG, so also check that there
     # are handlers instead of checking the log level
     if trace_log.handlers:
+        t0 = time.time()
```
Comment (oulgen) [inline torch/_logging/_internal.py:1088]: time.time uses floating point number, i'd use time.time_ns(), subtract them and then divide by 1000 to log ms number

**Review by oulgen — APPROVED** [review]
1) Can you log these to the events table I added as well? I assume you can gonna ninja this in there.
2) I thought we were more worried about torch logs not structured logs?

File: torch/_logging/_internal.py (hunk 1 above)
Comment (ezyang) [reply torch/_logging/_internal.py:1088]: It looks like Scuba prefers second timestamps, c.f. https://fb.workplace.com/groups/scuba.users/permalink/2499634333418500/

**Comment by jamesjwu** [comment]
I probably won't make it as detailed as every log message, will probably add it to dynamo_compile or something

**Pull request closed without merging**